src/utilities/core-utils.ts

```diff
@@ -134,6 +146,7 @@ export const CoreUtils = {
     objectToArray: _objectToArray,
     range: _.range,
     sleep: _sleep,
+    sleepSync: _sleepSync,
```

Comment:
While this is only being used in a single test now, we'll want to add some test coverage for it. Please at least add an issue to backfill tests

Reply:
will do